### DISCUSSION TITLE
[699130]World War Z

### DISCUSSION THREAD

**Discussion by fengjunziya** [pjy612/SteamManifestCache#140] [Share]
World War Z
```pid:699130```
秘钥:
```
699131;8e1c37df6ec32e30f09ea256f8732eb98b7955ec82c51c49960f0d078d057fbb
```
[699130_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13759826/699130_config.vdf.txt)
code:42114FF741B3EF61
email:1050128668@qq.com

**Comment by fengjunziya** [comment]
@pjy612 SAI新版本解压之后，内测码变动

**Reply by fengjunziya** [reply]
>new code: DC93899E9C2FD4D7

**Reply by pjy612** [reply]
查
